a10_octavia/tests/unit/controller/worker/tasks/test_virtual_port_tasks.py

```diff
@@ -412,3 +414,38 @@ def test_listener_update_for_pool_with_tcp_protocol(self, mock_protocol):
                                                                             listener.protocol_port,
                                                                             listener.default_pool_id
                                                                             )
+
+    def test_set_http_virtual_port_conn_limit_with_cli(self):
+        listener = self._mock_listener('HTTP', 1000)
+        flavor = {}
+        update_dict = {"connection_limit": 1000}
+
+        listener_task = task.ListenerUpdate()
+        listener_task.axapi_client = self.client_mock
+        self.conf.config(group=a10constants.LISTENER_CONF_SECTION,
+                         conn_limit=200)
+        listener_task.CONF = self.conf
+
+        with mock.patch('a10_octavia.common.openstack_mappings.virtual_port_protocol',
+                        return_value=listener.protocol):
+            listener_task.execute(LB, listener, VTHUNDER, flavor, update_dict)
+
+        args, kwargs = self.client_mock.slb.virtual_server.vport.replace.call_args
+        self.assertEqual(kwargs['conn_limit'], 1000)
+
+    def test_set_http_virtual_port_conn_limit_with_config(self):
+        listener = self._mock_listener('HTTP', 1000)
+        update_dict = {}
```

Comment:
Since this has been declared globally, why this is again declared here

Reply:
Removed this unwanted declaration